README.md

```diff
@@ -24,159 +29,17 @@ We're hiring great people to join our team in Copenhagen. Head over to [our jobs
 - `./playground`: source code of the playground
 - `./definitions.js`: constant definitions
 
-
 ## How can I help?
-You can help by:
-
-- Helping us find, add and improve integrations with 3rd party services
-- Helping us find, add and improve our Life Cycle Assesment / Carbon footprint data
-- Giving us ideas, feedback and reporting bugs
-
-### Integrating purchases and activities
-[We wrote a little article about what we believe makes a great integration](https://tmrow.slite.com/api/s/note/8LLSWazeBZZyS4BEQiLTnJ/What-makes-a-great-integration-for-Tomorrow).
-However, that shouldn't stop you from doing an integration which is interesting and useful to you! Our only requirement is that it can be quantified in greenhouse gas emissions.
-
-Integrations can rely on an API or even on scrapers if necessary.
-
-#### Suggesting an integration
-Here is the list of current 3rd party integrations:
-Official integrations:
-- ✈️ Tripit (tracks most airlines!)
-- ⚡ Barry
-- 🚗 Tesla Cockpit
-
-Community-supported integrations:
-- ✈️ Ryanair (contributor:[lauvrenn](https://github.com/lauvrenn))
-- ✈️ Wizzair (contributor:[lauvrenn](https://github.com/lauvrenn))
-- 🚂 Rejsekort
-- 🚂 Trainline (contributor:[liamgarrison](https://github.com/liamgarrison))
-- 🚂 Transport for London (contributor:[liamgarrison](https://github.com/liamgarrison))
-- ⚡ Sense (contributor:[snarfed](https://github.com/snarfed))
-- ⚡ Linky (contributor:[bokub](https://github.com/bokub))
-- ⚡ Ørsted (contributor:[felixdq](https://github.com/felixdq))
-- 📧 Outlook (contributor:[baywet](https://github.com/baywet))
-- 🚗 Renault Zoé
-- 🚗 Uber (contributor:[willtonkin](https://github.com/willtonkin))
-- 🚗 Automatic (contributor:[lauvrenn](https://github.com/lauvrenn))
-- 🚗 MinVolkswagen (contributor:[folkev0gn](https://github.com/folkev0gn))
-
-You can [suggest a new integration here](https://github.com/tmrowco/northapp-contrib/issues/new).
-
-#### Coding or debugging a new integration
```

Comment:
let's keep that section in case we wish to point developers to it

Reply:
Is it just this section or also the next one? 
https://github.com/tmrowco/northapp-contrib#how-to-make-an-integration-work